homu/server.py

```diff
@@ -438,6 +466,28 @@ def fail(err):
     return 'OK'
 
 
+def _try_create_deployment(state, repo_cfg):
```

Comment:
You are not expected to have known this, but "try" connotes a special kind of build within Mozilla. It will prevent future confusion if you name this method "_attempt_create_deployment" instead.